ingestion/src/metadata/profiler/metrics/static/null_count.py

```diff
@@ -50,9 +49,6 @@ def fn(self):
         """sqlalchemy function"""
         return SumFn(case([(column(self.col.name).is_(None), 1)], else_=0))
 
-    def df_fn(self, df=None):
+    def df_fn(self, dfs=None):
         """pandas function"""
-        from pandas import DataFrame  # pylint: disable=import-outside-toplevel
-
-        df = cast(DataFrame, df)
-        return df[self.col.name].isnull().sum()
+        return sum(map(lambda df: df[self.col.name].isnull().sum(), dfs))
```

Comment:
Generally `map` in Python has little use case. Most common is when you have already defined function
e.g.
```python
map(sum, [1,2,3,4])
```
But here we end up doing something a bit more convulated. It is more pythonic to do something like

```python
return sum(df[self.col.name].isnull().sum() for df in dfs)
```

Reply:
yes, I was thinking of a faster memory efficient approach, but yes, generators work as well.